Library/Formula/icecream.rb

```diff
@@ -0,0 +1,97 @@
+class Icecream < Formula
+  homepage "https://en.opensuse.org/Icecream"
+
+  option "without-docbook2X", "Build without man page"
+  option "enable-clang-wrappers", "Use symlink wrappers for clang/clang++"
+  option "enable-clang-rewrite-includes", "Use by default Clang's -frewrite-includes option"
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "docbook2X" => [:recommended, :build]
+
+  stable do
+    url "https://github.com/icecc/icecream/archive/1.0.1.tar.gz"
+    sha1 "2c3e43d34e4cbe8da5fa49e0518fd4db4c665f81"
+  end
+
+  devel do
+    url "https://github.com/icecc/icecream/archive/1.1rc1.tar.gz"
+    version "1.1rc1"
+    sha1 "ccde08f67297122710270b440492a528df042f52"
+  end
+
+  fails_with :clang do
+    build 600
+    cause "requires gnu libstdc++"
+  end
```

Comment:
Wouldn't ` ENV.libstdcxx if ENV.compiler == :clang ` work instead of ` fails_with :clang `?